### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio. [Document Types Page]

### ISSUE THREAD

**Issue by aj93222** [CMU-313/Mayan-EDMS#113]
There are multiple failing elements as low contrast text makes it difficult for users to read. The current lighthouse score for accessibility on the document types page is 89.

<img width="544" alt="Screen Shot 2021-09-08 at 10 56 18 PM" src="https://user-images.githubusercontent.com/89879976/132615015-6b7ab542-866d-4601-bf38-f05714aa70de.png">
